openfl/federated/plan/plan.py

```diff
@@ -29,7 +29,7 @@ class Plan(object):
     def Load(yaml_path: Path, default={}):
         """Load the plan from YAML file."""
         if yaml_path and yaml_path.exists():
-            return safe_load(yaml_path.read_text())
+            return yaml.load(yaml_path.read_text(), Loader=yaml.Loader)
```

Comment:
This should be changed back to `safe_load`

Reply:
`yaml.Loader` allows to serialize and deserialize Python objects like classes and functions. If we want to change some behavior (aggregation functions), we probably want to override a default function with a custom function by passing it to `override_config` keyword argument of `run_experiment` native method. This argument will replace a default plan setting with `!!python/name...` string that can only be deserialized via `yaml.Loader`. I wonder what was the point of `safe_load` here?

Reply:
I fixed this by skipping the dumping of the updated plan in the native API.